### ISSUE TITLE
bug: youtube continuous play even when should not

### ISSUE THREAD

**Issue by doublehp** [revanced/revanced-patches#2317]
### Type

Error at runtime

### Bug description

when reading some playlist, play continues on next video, even when continuous play icon is NOT-BOLD. In Vanced Kit, it took me months to understand that this icon can have multiple states, but it worked correctly. Since I have switched to ReVanced Manager, I have continuous play, whatever the state of the icon is bold or not (I do have loop play when the icon shows 1). This issue is by itself a sufficient reason for me to continue using Vanced Kit (even if it's hugely deprecated).

### Steps to reproduce

start playing a list, and press the continuous play button untill its un-bolded, and wait end of video.

### Relevant log output

```shell
none
```

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by oSumAtrIX** [comment]
This is default YouTube behavior

**Comment by doublehp** [comment]
Yes, and that's why I started using Vanced KIT; this feature is the MAIN reason for me to use VANCED. The patch is available in Vanced KIT (the old deprecated project); ReVanced Manager obviously failed to apply this patch.

But since 2 weeks, playlists are broken in Vanced Kit; Youtube seems to have changed AGAIN their API.

This behaviour is also the default in Netflix and Prime, and, thats why I stopped using them. If ReVanced does not fix this, I may also stop using Youtube.
